### ISSUE TITLE
Button: focus indicator does not meet color contrast needs

### ISSUE THREAD

**Issue by danethurber** [pluralsight/design-system#1586]
![Screen Shot 2021-01-29 at 9 36 56 AM](https://user-images.githubusercontent.com/226356/106301971-97782200-6215-11eb-967a-c12cd4d3952b.png)

**Comment by jaketrent** [comment]
Related: #1587

**Comment by sweeting** [comment]
Closed by #1593
